docs/plugins/woodpecker-plugins/plugins.json

```diff
@@ -148,6 +148,11 @@
       "name": "NixOS Remote Builder",
       "docs": "https://codeberg.org/JohnWalkerx/nix-remote-builder-plugin/raw/branch/main/docs.md",
       "verified": false
+    },
+    {
+      "name": "Release Helper",
+      "docs": "https://github.com/woodpecker-ci/plugin-ready-release-go/blob/main/README.md",
```

Comment:
Should be a special `docs.md` file containing a table with some details at the top.